engine/table/src/main/java/io/deephaven/engine/table/impl/QueryTable.java

```diff
@@ -1266,7 +1267,10 @@ private Table selectOrUpdate(Flavor flavor, final SelectColumn... selectColumns)
 
                     final QueryTable resultTable;
                     final LivenessScope liveResultCapture = isRefreshing() ? new LivenessScope() : null;
-                    try (final SafeCloseable ignored = liveResultCapture != null ? liveResultCapture::release : null) {
+                    try (final SafeCloseable ignored1 = liveResultCapture != null ? liveResultCapture::release : null;
+                            final SafeCloseable ignored2 = ExecutionContext.getDefaultContext().open()) {
+                        // we open the default context here to ensure that the update processing happens in the default
```

Comment:
This is to save the user from himself. If the formula column needs access to the ExecutionContext it should be installed by the user from within his formula. By opening the default poisoned context, we guarantee that the `fakeUpdate` pass through the formula is invoked without any user installed context nor any systemic context.

Note that the context installed at this point in time is not the context that will be installed from a view column (maybe we should install the default poisoned execution context inside of ConstructSnapshot, too). As discussed in chat, it might be appropriate to back-out the change for update graph notifications to specify their execution context; the user is supposed to install the execution context from within their formula.